serverless-functions/src/functions/features/park-interaction/common/unpark-interaction.js

```diff
@@ -44,6 +46,8 @@ exports.handler = prepareStudioFunction(requiredParameters, async (context, even
         properties: {
           workspace_sid: context.TWILIO_FLEX_WORKSPACE_SID,
           workflow_sid: workflowSid,
+          queue_sid: queueSid,
+          worker_sid: workerSid,
```

Comment:
The routing models for unparking might be different when using the interactions list versus customer-initiated unparking. For example, when unparking from the list I would want the interaction to route to me, but when the customer unparks by sending a reply, they might want it to go to any available worker. Maybe this should be configurable, or maybe unparks from the list should have different routing behavior? (paging @jhunter-twilio)

Reply:
I also think we should remove the conversation from the Sync map when it is unparked.

Reply:
2 cents here:
For this unpark feature routing model, the task/interaction should be routed to same agent.
For customer-initiated unparking that could route to any available agent, as it is today, if the customer wants to change that behavior, they can use something as Known Agent Routing (preferred agent) but they may have to review/change their TR workflow to use it.

Reply:
I think it makes sense to have it separated:
1. If customer-initiated unparking: go to any available worker
2. If agent-initiated unparking: route to the agent directly

I'll update this function to check the source of unparking, and add `queue_sid` and `worker_sid` conditionally

Reply:
Implemented this in this commit [61d39aa](https://github.com/twilio-professional-services/flex-project-template/pull/337/commits/61d39aaf1cc5becafaff1a4e11b15c1fce68b63a)